code/src/index.css

```diff
@@ -1,3 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```

Comment:
In the design specs it was specified to use Helvetica. See below:
- Album title - 14px Helvetica #ffffff
- Artist name - 14px Helvetica #a0a0a0